ginga/aggw/ImageViewAgg.py

```diff
@@ -20,56 +20,11 @@ def __init__(self, logger=None, rgbmap=None, settings=None):
 
         self.renderer = CanvasRenderer(self)
 
-    def get_surface(self):
-        return self.surface
-
-    def configure_surface(self, width, height):
-        # tell renderer about our new size
-        self.renderer.resize((width, height))
-
-        # inform the base class about the actual window size
-        self.configure(width, height)
-
-    def get_image_as_array(self):
-        # TO BE DEPRECATED: DO NOT USE
-        return self.renderer.get_surface_as_array()
-
-    def get_image_as_buffer(self, output=None):
-        # TO BE DEPRECATED: DO NOT USE
-        return self.renderer.get_surface_as_buffer()
-
-    def get_rgb_image_as_buffer(self, output=None, format='png', quality=90):
-        # TO BE DEPRECATED: DO NOT USE
-        return self.renderer.get_surface_as_rgb_format_buffer(output=output,
-                                                              format=format,
-                                                              quality=quality)
-
-    def get_rgb_image_as_bytes(self, format='png', quality=90):
-        # TO BE DEPRECATED: DO NOT USE
-        return self.renderer.get_surface_as_rgb_format_bytes(format=format,
-                                                             quality=quality)
-
-    def save_rgb_image_as_file(self, filepath, format='png', quality=90):
-        # TO BE DEPRECATED: DO NOT USE
```

Comment:
Does this mean deprecation is no longer planned?

Reply:
Yes, at least for now, and the comment is removed where the method migrated to.